src/app/workbasket/filters/workbasket-filters.component.spec.ts

```diff
@@ -1,1058 +0,0 @@
-          description: ''
-        }
-      ],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    }
-  ];
-
-  const JURISDICTION_2: Jurisdiction = {
-    id: 'J2',
-    name: 'Jurisdiction 2',
-    description: '',
-    caseTypes: []
-  };
-
-  const CASE_TYPES_2: CaseType[] = [
-    {
-      id: 'CT1',
-      name: 'Case type 1',
-      description: '',
-      states: [],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    },
-    {
-      id: 'CT2',
-      name: 'Case type 2',
-      description: '',
-      states: [
-        {
-          id: 'S1',
-          name: 'State 1',
-          description: ''
-        },
-        {
-          id: 'S2',
-          name: 'State 2',
-          description: ''
-        }
-      ],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    },
-    {
-      id: 'CT3',
-      name: 'Case type 3',
-      description: '',
-      states: [],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    }
-  ];
-
-  const DEFAULT_CASE_TYPE = CASE_TYPES_2[1];
-  const DEFAULT_CASE_STATE = DEFAULT_CASE_TYPE.states[1];
-
-  const CRUD_FILTERED_CASE_TYPES: CaseType[] = [
-    {
-      id: 'CT1',
-      name: 'Case type 1',
-      description: '',
-      states: [{
-        id: 'S1',
-        name: 'State 1',
-        description: ''
-      }],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    },
-    {
-      id: 'CT3',
-      name: 'Case type 3',
-      description: '',
-      states: [{
-        id: 'S1',
-        name: 'State 1',
-        description: ''
-      }],
-      events: [],
-      case_fields: [],
-      jurisdiction: null
-    }
-  ];
-  const EMPTY_CASE_TYPES: CaseType[] = [];
-  const CASE_TYPE_WITH_EMPTY_STATES: CaseType[] = [{
-    id: 'CT3',
-    name: 'Case type 3',
-    description: '',
-    states: [],
-    events: [],
-    case_fields: [],
-    jurisdiction: null
-  }];
-
-  let createWorkbasketInputs = () => {
-    return [
-      createWBInput('Label 1', 1, 'PersonFirstName', 'Text', 'Text', '', 'Mohammed', false),
-      createWBInput('Label 2', 2, 'PersonLastName', 'Text', 'Text', '', 'Khatri', true),
-      createWBInput('Label 3', 3, 'PersonAddress', 'Text', 'Text', 'Country', '', false)
-    ]
-  };
-
-  const TEST_WORKBASKET_INPUTS: WorkbasketInputModel[] = createWorkbasketInputs();
-
-  const METADATA_FIELDS = ['PersonLastName'];
-
-  const $APPLY_BUTTON = By.css('button');
-
-  let fixture: ComponentFixture<WorkbasketFiltersComponent>;
-  let component: WorkbasketFiltersComponent;
-  let de: DebugElement;
-
-  let workbasketHandler;
-  let router: any;
-  let activatedRoute: any;
-  let jurisdictionService: JurisdictionService;
-  let workbasketInputFilterService: any;
-  let orderService: any;
-  let alertService: AlertService;
-  let windowService;
-  const TEST_FORM_GROUP = new FormGroup({});
-
-  describe('Clear localStorage for workbasket filters', () => {
```

Comment:
These tests are removed here and not added in the ui toolkit for case-list-filters.components.ts. Are we getting rid of them?